File: week1/mario-more/mario.c
// Asks for height & makes pyramids similar to the ones in Super Mario Bros.

#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // i,j,k,l are counter variables
    int height, i, j, k, l;

    // User prompt for height
    do
    {
        height = get_int("How tall should the pyramid be?   ");
    }
    while (height > 8 || height <= 0);


    // Loop per horiz line
    for (i = 0; i < height; i++)
    {
        // Blank space in the left pyramid
        for (j = (height - i - 1); j > 0;  j--)
        {
            printf(" ");
        }

        // Blocks in the left pyramid
        for (k = 0; k <= i; k++)
        {
            printf("#");
        }

        // Double space between pyramids
        printf("  ");

        // Blocks in the right pyramid
        for (l = 0; l <= i;  l++)
        {
            printf("#");
        }

        // Go to next line
        printf("\n");

    }
}

File: week6/sentimental-mario-more/mario.py
# Print out a double half-pyrmamid of a specific height, similar to pyramids in Mario
# TODO
import cs50

# Keep on asking until a valid height was given
while True:
    height = cs50.get_int("Height: ")
    if (height >= 1) and (height <= 8):
        break

# Loop to each row
for i in range(height):
    # Left pyramid
    # Print white spaces
    for j in range(height - i - 1):
        print(" ", end="")
    # Print hashes
    for k in range(i + 1):
        print("#", end="")

    # Double white spaces between the pyramids
    print("  ", end="")

    # Right pyramid
    # Print hashes
    for l in range(i + 1):
        print("#", end="")

    # Go to next line
    print()